spec.bs

```diff
@@ -229,6 +239,8 @@ const publicKey = {
   rp: {
     name: "Fancy Bank",
   },
+  
+  Q: Should we perhaps require that  rpId (domain) must be specified if this is inside an iFrame? For top level it's probably OK to use the top level domain?
```

Comment:
I'm not immediately sure why one would need to specify rpId here. I believe the WebAuthn spec says to use the callers (iframe's) domain:

https://w3c.github.io/webauthn/#dom-publickeycredentialcreationoptions-rp
```
Its value’s id member specifies the RP ID the credential should be scoped to. If omitted, its value
will be the CredentialsContainer object’s relevant settings object's origin's effective domain. See
§ 5.4.2 Relying Party Parameters for Credential Generation (dictionary PublicKeyCredentialRpEntity)
for further details.
```

Have I overlooked something?

Reply:
Given that this is a Registration and there is no cross domain and no custom display, I agree that this should be fine.